glutin/src/application.rs

```diff
@@ -65,8 +65,8 @@ where
     };
 
     let context = {
-        let builder = settings.window.into_builder(
-            &application.title(),
+        let builder = settings.window.clone().into_builder(
+            &settings.window.title,
```

Comment:
Since the `title` is in the `settings.window` already, we can just remove the argument here and access it directly in `into_builder`.